app/liquid_tags/kotlin_tag.rb

```diff
@@ -1,56 +1,47 @@
 class KotlinTag < LiquidTagBase
   PARTIAL = "liquids/kotlin".freeze
-  PARAM_REGEXP = /\A[a-zA-Z0-9]+\z/
+  REGISTRY_REGEXP = %r{https://pl.kotl.in/(?<id>[\w-]+)(?:\?)?(?<params>[\w=&]+)?}
```

Comment:
should this include play.kotlinlang.org or will users always need to use the short domain name?

Reply:
Interesting question, Dan.
For sandboxes on the Kotlin website, the URL is of the format:
```
https://play.kotlinlang.org/#eyJ2ZXJzaW9uIjoiMS42LjEwIiwiY29kZSI6InBhY2thZ2Ugb3JnLmtvdGxpbmxhbmcucGxheSAgICAgICAgIC8vIDFcblxuZnVuIG1haW4oKSB7ICAgICAgICAgICAgICAgICAgICAgICAgLy8gMlxuICAgIHByaW50bG4oXCJIZWxsbywgV29ybGQhXCIpICAgICAgICAvLyAzXG4gICAgcHJpbnRsbihcIkJPT09PT09cIilcbn0iLCJwbGF0Zm9ybSI6ImphdmEiLCJhcmdzIjoiIn0=
```
So that uses the long domain. However, all Share URL formats (Link, Embed and Medium) use the short domain name `pl.kotl.in`.
I also ran a pretty good search on Blazer for articles or comments that used the Kotlin embed and all embeds used the short domain as well.